journals/apps/journals/models.py

```diff
@@ -309,38 +310,36 @@ def generate_require_auth_basket_url(self, sku):
         encoded_basket_url = quote(basket_url)
         return "/require_auth?forward={}".format(encoded_basket_url)
 
-
     def generate_basket_url(self, sku):
         ecommerce_base_url = self.site.siteconfiguration.ecommerce_public_url_root
         (scheme, netloc, _, _, _) = urlsplit(ecommerce_base_url)
         basket_url = urlunsplit((
             scheme,
             netloc,
             '/basket/add/',
-            f'sku={sku}',
+            'sku={sku}',
```

Comment:
This is an f-string, you can't just remove the `f`. It's a feature we can use since we're in Python 3.6. Saves a bunch of `.format()`s. If you're curious, see PEP 498 for more info.